src/event-cache/EventCache.ts

```diff
@@ -8,6 +8,10 @@ import {
     UserDetails,
     RumEvent
 } from '../dispatch/dataplane';
+import { INSTALL_SCRIPT, INSTALL_MODULE } from '../utils/constants';
+
+import packageInfo from '../../package.json';
```

Comment:
issue: package.json is pretty large to be including in the JS bundle. Is there a way we could automatically create a minimal 'version.json' file at build time?

Reply:
Using the `GenerateJsonPlugin` webpack plugin, was able to create a separate `version.json` file at build time